src/overlays/actors/ovl_Fishing/z_fishing.c

```diff
@@ -1625,24 +1638,24 @@ void Fishing_DrawLureHook(PlayState* play, Vec3f* pos, Vec3f* refPos, u8 hookInd
               G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
     gSPDisplayList(POLY_OPA_DISP++, gFishingLureHookDL);
 
-    if ((hookIndex == 1) && (D_80B7A68C != 0)) {
+    if ((hookIndex == 1) && (sOwnerHatTheft != 0)) {
         Matrix_Scale(2.0f, 2.0f, 2.0f, MTXMODE_APPLY);
         Matrix_Translate(250.0f, 0.0f, -1400.0f, MTXMODE_APPLY);
         Matrix_Push();
 
-        if (D_80B7A690 != 0) {
+        if (sOwnerHatHooked != 0) {
```

Comment:
fyi, for example `sOwnerHatHooked` is only ever compared to 0, and set to 0/1
in these cases (booleans) we use false/true instead of 0/1
and typically `if (sOwnerHatHooked != 0)` can be replaced with `if (sOwnerHatHooked)`
(and `if (sOwnerHatHooked == 0)` with `if (!sOwnerHatHooked)` if needed)
and the rom should still match. Usually it does but sometimes it doesn't, your mileage may vary

Reply:
if the variable is boolean it should be prefixed with `is` or `has`, e.g. `sIsOwnerHatHooked`

Additionally, I'm not too keen on the variable name here, since it refers to the hat as being `OwnerHat`.